.github/workflows/verify.yml

```diff
@@ -97,13 +97,15 @@ jobs:
       - run: make check-config-template
 
   get-script:
+    if: ${{ github.base_ref == 'main' }}
```

Comment:
Does it also run for PR's targeting `main` now?

Reply:
Yes.